xml/System.Windows.Forms/ListViewItemStateImageIndexConverter.xml

```diff
@@ -76,8 +76,9 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether a <see langword="none" /> or <see langword="null" /> value is valid in the <see cref="T:System.ComponentModel.TypeConverter.StandardValuesCollection" /> collection.</summary>
```

Comment:
`...whether a <see langword="none" />..` - `none` is not a langword.
Should it be `<see cref="none" />`?

Reply:
I could wrap it with <c> instead. see cref links are for API links, which is not the case here.

Reply:
What is a none or a none value?

Reply:
Looking at this: https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.imagekeyconverter.includenoneasstandardvalue?view=netframework-4.8

I would say that 'none' is the display name that is used when standard values are presented in the UI such as a propertygrid drop down. I would not use any markdown around 'none', but we need markdown around 'null'. Or we can go with what they did in the link that I quoted  above, only say `null` and don't mark it as a keyword.

Reply:
I prefer the version from the imagekeyconverter and maybe add some info in the remarks to how the null value maps to the none in the display name. I'll do it like that then. I'll open a new PR to fix that in similar APIs as well.

Reply:
```suggestion
        <summary>Gets a value that indicates whether a <see langword="null" /> value is valid in the <see cref="T:System.ComponentModel.TypeConverter.StandardValuesCollection" /> collection.</summary>
```